docs/GettingStartedDocs/using_oe_sdk.md

```diff
@@ -17,12 +19,13 @@ By default, the Open Enclave SDK is installed to `/opt/openenclave`. It contains
 | lib/openenclave/host         | Library for linking into the host process of the enclave. |
 | lib/openenclave/debugger     | Libraries used by the gdb plug-in for debugging enclaves. |
 | share/openenclave/samples    | Sample code showing how to use the Open Enclave SDK. |
-| share/pkgconfig              | Pkg-config files for header and library includes when building Open Enclave apps. |
 
-## Configure environment variables for Open Enclave SDK
+On Linux, the Open Enclave SDK installation also contains the following folder:
+| share/pkgconfig              | Pkg-config files for header and library includes when building Open Enclave apps. |
 
+## Configure environment variables for Open Enclave SDK for Linux
 For ease of development, we recommend adding:
-- Open Enclave SDK `bin` folder to `PATH`, for use of our tools (such as `oegdb` and `oeedger8r`).
+- Open Enclave SDK `bin` folder to `PATH`, for use of our tools (such as `oegdb`, and `oeedger8r`).
```

Comment:
Oxford comma after `oegdb` is unnecessary here, as are the parentheses around the examples of the two tools.